src/index.js

```diff
@@ -265,6 +265,40 @@ class HCaptcha extends React.Component {
       return hcaptcha.execute(captchaId, opts);
     }
 
+    setData (opts) {
+      const { captchaId } = this.state;
+
+      if (!this.isReady()) {
+        return;
+      }
+
+      if (opts && typeof opts !== "object") {
+        opts = null;
+      }
+
+      hcaptcha.setData(captchaId, opts);
+    }
+
+    getResponse() {
+      const { captchaId } = this.state;
+
+      if (!this.isReady()) {
+        return;
+      }
+      //Get response token from hCaptcha widget
+      return hcaptcha.getResponse(captchaId);
+    }
+
+    getRespKey() {
+      const { captchaId } = this.state;
+
+      if (!this.isReady()) {
+        return;
+      }
+      //Get current challenge session id from hCaptcha widget
+      return hcaptcha.getRespKey(captchaId)  
```

Comment:
Code can be streamlined a bit:

```js
getResponse() {
  return hcaptcha.getResponse(this.state.captchaId);
}

getRespKey() {
  return hcaptcha.getRespKey(this.state.captchaId)
}
```

Reply:
Done https://github.com/hCaptcha/react-hcaptcha/pull/122/commits/09a2cab4b5e9dae2f5543b4e5839100ad0618c5a